WcaOnRails/vendor/assets/javascripts/jquery.wca-autocomplete.js

```diff
@@ -15,11 +15,12 @@
       var posts_search = $(that).hasClass("wca-autocomplete-posts_search");
 
       var delimiter = ',';
-      var searchFields = [];
-      searchFields = searchFields.concat([ 'wca_id', 'name' ]); // user search fields
-      searchFields = searchFields.concat([ 'id', 'name', 'cellName', 'cityName', 'countryId' ]); // competition search fields
-      searchFields = searchFields.concat([ 'title', 'body' ]); // post search fields
-      searchFields = searchFields.concat([ 'text' ]); // regulation search fields ('id' already added)
+      var searchFields = [
+        'wca_id', 'name', // user search fields
+        'id', 'name', 'cellName', 'cityName', 'countryId', // competition search fields
```

Comment:
Actually I've just notice that name appears twice here. As you haven't duplicated `id` (the lase comment) we should be consistent and do the same here =)